content/en/agent/autodiscovery/template_variables.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Autodiscovery Template Variables
+kind: documentation
+disable_toc: true
+further_reading:
+- link: "/agent/autodiscovery/integrations"
+  tag: "Documentation"
+  text: "Create and load an Autodiscovery Integration Template"
+- link: "/agent/autodiscovery/ad_identifiers"
+  tag: "Documentation"
+  text: "Match a container with the corresponding Integration Template"
+- link: "/agent/autodiscovery/management"
+  tag: "Documentation"
+  text: "Manage which Container to include in the Agent Autodiscovery"
+---
+
+Use the following Template Variables when configuring Autodiscovery in order to dynamically assign your containers values:
+
+
+| Template Variable           | Description                                                                                                                                                                                               |
+| --------------------------  | ---                                                                                                                                                                                                       |
+| `"%%host%%"`                | Auto-detects the network. For single-network containers, it returns its corresponding IP.                                                                                                                 |
+| `"%%host_<NETWORK NAME>%%"` | Specifies the network name to use, when attached to multiple networks                                                                                                                                     |
+| `"%%port%%"`                | Uses the highest exposed port **sorted numerically and in ascending order**,<br> e.g. it would return `8443` for a container that exposes ports `80`, `443`, and `8443`                                   |
+| `"%%port_<NUMBER_X>%%"`     | Uses the `<NUMBER_X>` port **sorted numerically and in ascending order**,<br> e.g. if a container exposes ports `80`, `443`, and `8443`, `"%%port_0%%` refers to port `80`, `"%%port_1%%"` refers to `443` |
```

Comment:
we also support `%%port_<NAME>%%` see https://github.com/DataDog/datadog-agent/pull/1754